### PR TITLE
PR #1000

### PR DESCRIPTION
**Pull request by michael-grace**
Realistically, I did this just to be PR #1000. We don't actaully need to merge this until we're ready to 😛

### CODE REVIEW

**Review by mstratford — APPROVED** [review]
It's time.

File: src/Menus/menu.json [hunk 1]
```diff
@@ -118,9 +118,6 @@
                             "title": "WebStudio",
                             "url": "module=MyRadio,action=webstudio",
                             "description": "Access WebStudio. Plan, create, practise and broadcast your shows!"
-                        },
-                        {
-                            "template": "MyRadio/showPlanner.twig"
```
Comment (markspolakovs) [inline src/Menus/menu.json:123]: Should we delete this template, so we don’t have loose files lying around?

Comment (mstratford) [reply src/Menus/menu.json:123]: There's a load of dead showplanner templates etc that could be chucked. Back from the days when it was attempted to control BAPS2

Comment (markspolakovs) [reply src/Menus/menu.json:123]: Eh, I'll file a separate issue for that to not block this.

**Pull request merged**